lldb/unittests/Core/FormatEntityTest.cpp

```diff
@@ -8,16 +8,36 @@
 
 #include "lldb/Core/FormatEntity.h"
 #include "lldb/Utility/Status.h"
-
+#include "lldb/Utility/StreamString.h"
 #include "llvm/ADT/StringRef.h"
+#include "llvm/Support/Error.h"
+#include "llvm/Testing/Support/Error.h"
 #include "gtest/gtest.h"
 
 using namespace lldb_private;
+using namespace llvm;
 
 using Definition = FormatEntity::Entry::Definition;
 using Entry = FormatEntity::Entry;
 
-TEST(FormatEntityTest, DefinitionConstructionNameAndType) {
+namespace {
+class FormatEntityTest : public ::testing::Test {
+public:
+  Expected<std::string> Format(StringRef format_str) {
```

Comment:
As there's (still) no state, you could just make this a static helper function.

```suggestion
static  Expected<std::string> Format(StringRef format_str) {
```

Reply:
Sounds good. Originally I had an `ASSERT` in there but I went with the expected approach so the failure is attributed to the correct line.